### PR TITLE
created new pr to deal with issues from previous branch

### PR DESCRIPTION
**Pull request by KaiElwood**
Had to fix the issues associated with my earlier branch. Hopefully this works. The one issue I'm finding is that it only works when the view is in the inspect element view –– something about line 15 "<main id="site-content" role="main">" in site-content.php is throwing everything off. Not sure what the issue is there. Will work on it tomorrow.

#46

### CODE REVIEW

**Comment by KaiElwood** [comment]
"<main id="site-content" role="main">"

**Comment by KaiElwood** [comment]
the space between the quotes should say <main id="site-content" role="main">

**Review by jnschrag — CHANGES_REQUESTED** [review]
We talked through the requested changes in our pairing session today. :) Gonna do some reworking on the semantic HTML, naming, and making things a little more future proof.

File: wp-content/themes/csisnuclearnetwork/assets/_scss/base/_base.scss [hunk 1]
```diff
@@ -26,7 +28,7 @@
 
 html,
 body {
-  // overflow-x: hidden;
+  overflow-x: hidden;
```
Comment (jnschrag) [inline wp-content/themes/csisnuclearnetwork/assets/_scss/base/_base.scss:31]: Why was this added? Typically we try to avoid adding an `overflow: hidden` to the body because it breaks things like `position: sticky` which are used for other elements on the page.

File: wp-content/themes/csisnuclearnetwork/assets/_scss/components/_home-about-poni.scss [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+    @include full-width-background($color-blue-000);
+
+    @include breakpoint('large') {
+      flex-wrap: nowrap;
+      gap: rem(32px);
+    }
+
+    &::before,
+    &::after {
+      @include z-index('postBG');
```
Comment (jnschrag) [inline wp-content/themes/csisnuclearnetwork/assets/_scss/components/_home-about-poni.scss:19]: Instead of using `postBG` here, it's better practice to add `homeAboutPONI` as an entry in the `z-index` map in the variables file. That way if changes need to be made to the `postBG` index, it doesn't have unexpected side effects on this part of the site where we wouldn't expect it.

File: wp-content/themes/csisnuclearnetwork/assets/_scss/components/_home-about-poni.scss [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+      &:first-of-type {
+        padding-bottom: rem(24);
+      }
+      
+      &-image {
+        grid-area: image;
+
+        /* stylelint-disable-next-line */
+        img {
+          width: rem(100);
```
Comment (jnschrag) [inline wp-content/themes/csisnuclearnetwork/assets/_scss/components/_home-about-poni.scss:93]: On images, we typically do want to keep `px` for the unit values.

File: wp-content/themes/csisnuclearnetwork/assets/_scss/components/_home-about-poni.scss [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+
+      &-bio {
+        grid-area: bio;
+        color: $color-white-165;
+        @extend %text-style-label-small;
+      }
+    }
+
+    &-familyphoto {
+      // NOT WORKING
```
Comment (jnschrag) [inline wp-content/themes/csisnuclearnetwork/assets/_scss/components/_home-about-poni.scss:113]: Remove these comments now that you don't need them.

File: wp-content/themes/csisnuclearnetwork/assets/_scss/components/_home-about-poni.scss [hunk 5]
```diff
@@ -0,0 +1,147 @@
...
+        padding-left: var(--container-padding);
+        color: $color-white-165;
+        @extend %text-style-paragraph-medium-short;
+
+        @include breakpoint('large') {
+          padding-left: 0;
+        }
+      }
+
+      @include breakpoint('large') {
```
Comment (jnschrag) [inline wp-content/themes/csisnuclearnetwork/assets/_scss/components/_home-about-poni.scss:135]: If these are meant to apply to the familyphoto class, move them so they're after the base styles for `&-familyphoto` and before the `img` styles.

**Review by jnschrag — CHANGES_REQUESTED** [review]
Nice work on this, Kai. It's much improved! There's a few additional things that can still be tweaked, but I think this is almost ready to go. I've left comments on specific lines, but have a few comments here too.

On the `&-people` class, you're using Flexbox to control the flow of the items in that list. That works well, but we can actually simplify some of the existing code a little bit and make it more fluid. I've included the relevant CSS below with comments to explain what is happening. Please remove the comments before adding it to the branch.

```
&-people {
  display: flex;
  flex-wrap: wrap;
  gap: rem(24); // You should be able to remove the &:first-of-type padding-bottom: rem(24) with this
  padding-bottom: rem(40);
}

&-person {
  display: grid;
  grid-template-areas: 
    'image title'
    'image bio';
  grid-template-columns: 100px auto;
  grid-template-rows: min-content auto;
  flex: 1 1 calc(50% - #{rem(24)}); // The `flex` property is applied on the children of a flex container and is shorthand for the `flex-grow`, `flex-shrink`, and `flex-basis` properties. The `1 1` part of this are telling it that we want the items to grow and/or shrink to fill the space as needed. The `calc(50% - #{rem(24)})` sets the approximate width of each size of each item to 50%, minus the size of the gap. Because the flex-grow & flex-shrink properties are set to 1, the items will automatically grow/shrink as the screen size changes.
  column-gap: rem(16);
  row-gap: rem(4);

  // Remove all code related to setting the width & padding properties on the `&-person`.
}
```

File: wp-content/themes/csisnuclearnetwork/assets/_scss/base/_base.scss (hunk 1 above)
Comment (KaiElwood) [reply wp-content/themes/csisnuclearnetwork/assets/_scss/base/_base.scss:31]: Grid-template-rows: MAX-CONTENT (edit)

**Review by jnschrag — APPROVED** [review]
Looks good, Kai!

@urchykli This is good to merge in. You'll probably need to pull in from dev to see the changes. There's a little bit of wonkiness at larger screen sizes that I think will be resolved when the rest of the home page is finished.

I wanted to flag that Kai and I abstracted out a `.home__subtitle` and `.home__subtitle--border` class for the subtitles that have the half-border underneath them. The color of the text & bar can be modified with CSS variables. So those classes can be used in other components of the home page.
